Makefile

```diff
@@ -110,3 +110,15 @@ run-local: build-local-container load-image deploy ## Run Combo on local environ
 run-e2e: run test-e2e ## Run Combo and trigger the e2e tests for it
 
 run-e2e-local: run-local test-e2e ## Run Combo on local environment and trigger e2e tests for it using Dockerfile.local
+
+##################
+# Release Targets#
+##################
+export DISABLE_RELEASE_PIPELINE ?= true
+substitute:
+	envsubst < .goreleaser.template.yml > .goreleaser.yml
+
+release: GORELEASER ?= goreleaser
+release: GORELEASER_ARGS ?= --snapshot --rm-dist
+release: goreleaser substitute
```

Comment:
I think we'd need to remove the `goreleaser` target as a prerequisite target as that target doesn't appear to exist?

Reply:
Oops... missed that in the copy/paste clean up.